common/util/http_util.go

```diff
@@ -235,7 +239,7 @@ func HTTPWithHeaders(method, url string, headers map[string]string, timeout time
 		req.Header.Add(k, v)
 	}
 
-	return http.DefaultClient.Do(req)
+	return DefaultHTTPSClient.Do(req)
```

Comment:
In fact, the `HTTPGet` will call this function finally. And `dfget` and `dfdaemon` will also use the function `HTTPGet`.